README.md

```diff
@@ -1,5 +1,5 @@
 # NuGet2
-This is the home of nuget.core and all the repos from codeplex.
+This is the home of nuget.core and all the repos from codeplex. This repository is for Version 2 of NuGet. Version 3 of the Nuget client library has moved to [Nuget.Client](https://github.com/NuGet/nuget.client/). See [this blog post](http://daveaglick.com/posts/exploring-the-nuget-v3-libraries-part-1) to get an overview of the changes between version 2 and 3.
```

Comment:
Please make the change to point to this URL - http://nugetdocsbeta.azurewebsites.net/ndocs/api/nuget-api-v3